src/services/doppler-mvc-client.ts

```diff
@@ -0,0 +1,34 @@
+import { AxiosInstance, AxiosStatic } from 'axios';
+
+// TODO: Consider better the name, DopplerMvc tries to make reference to Doppler API implemented
```

Comment:
Maybe rename `DopplerMvcClient` to  `DopplerAppClient`? What do you think?

Reply:
Nice!, it is simple and correct, I do not know how I did not think it before, thanks!!

Reply:
Mmm, no, because App could make reference to current WebApp. I want to make reference to the old Doppler API hosted by MVC. Another idea?

Reply:
`old Doppler API` not soap api right? O_o Why not `dopplerApiClient` then?

Reply:
> `old Doppler API` not soap api right? O_o Why not `dopplerApiClient` then?

Not SOAP, not REST, it is old doppler MVC backend, for that reason I am not comfortable with `dopplerApiClient`

Reply:
We finally choose `DopplerLegacyClient`.